src/workerd/api/actor-state.h

```diff
@@ -475,7 +475,9 @@ class DurableObjectState: public jsg::Object {
   kj::Array<kj::StringPtr> getTags(jsg::Lock& js, jsg::Ref<api::WebSocket> ws);
 
   JSG_RESOURCE_TYPE(DurableObjectState, CompatibilityFlags::Reader flags) {
-    JSG_METHOD(waitUntil);
+    if (!flags.getNoDurableObjectWaitUntil()) {
```

Comment:
nit: it would probably be helpful to also add a comment to the implementation of `waitUntil` in `actor-state.c++` indicating that it is deprecated